examples/pipelines/burnout_on_video.pipe.in

```diff
@@ -0,0 +1,72 @@
+# ================================================================
+process input
+  :: video_input
+  video_filename   	= @EXAMPLE_DIR@/videos/drone-video.mp4
+  frame_time        	= .3333
+
+  video_reader:type	= vidl_ffmpeg
+
+# ================================================================
+process filter
+  ::image_filter
+  filter:type                    	= vxl_pixel_feature_extractor
```

Comment:
Where is this? Is this a PR that hasn't landed yet?

Reply:
This is in #1198. This also depends on #1206 and #1207. The integration branch, #1210, pulls everything together.